stingray/utils.py

```diff
@@ -360,18 +368,29 @@ def contiguous_regions(condition):
 
 
 def is_int(obj):
+    """Test if object is an integer."""
     return isinstance(obj, (numbers.Integral, np.integer))
 
 
-def get_random_state(random_state=None):
-    if not random_state:
+def get_random_state(seed=None):
+    """Return a Mersenne Twister pseudo-random number generator.
+
+    Parameters
+    ----------
+    seed : integer, optional, default None
+
+    Returns
+    -------
+    random_state : mtrand.RandomState object
+    """
+    if not seed:
```

Comment:
@swapsha96: This variable was called `random_state` exactly because it not only accepts a integer `seed`, but it could be either an `int` or a random state object. This is why tests are failing!
Please modify the name of the variable back, and update the docstrings to account for this case. 
It's very good that you started this docstring review, because evidently this function needed to be documented properly! Sorry about that, and thanks

Reply:
I should be more careful with the changes. I'll make sure that I ask first if I am not sure about something. I think proper documentation is as important as adding new features. Thanks! I'll do that.